src/hip/hip_build_utils.cpp

```diff
@@ -140,6 +146,31 @@ boost::filesystem::path HipBuild(boost::optional<TmpDir>& tmp_dir,
         return hsaco->path();
     }
     else
+#endif
+#ifdef MIOPEN_OFFLOADBUNDLER_BIN
```

Comment:
There may be a scenario where both extractkernel bin and offloadbunder is found. Because HIP-Clang does support both options. Should we use else if here?

Reply:
The CMakeList.txt should filter that case out.